inc/RestCommand.php

```diff
@@ -61,9 +61,15 @@ public function set_auth( $auth ) {
 	public function create_item( $args, $assoc_args ) {
 		list( $status, $body ) = $this->do_request( 'POST', $this->get_base_route(), $assoc_args );
 		if ( Utils\get_flag_value( $assoc_args, 'porcelain' ) ) {
-			WP_CLI::line( $body['id'] );
+			if ( $status < 400 ) {
```

Comment:
Would you please add apache to the travis config? It would likely much churn for me to get that working. The docs to do it are at https://docs.travis-ci.com/user/languages/php/#Apache-%2B-PHP
Why? None of the test cases use the --http code path. They are all calling locally. The code paths in restful are significantly different between them. And the results coming back from REST are sometimes different. I encountered this during my testing and handled the differences in code.
With apache in place, I can add test cases which exercise the new PRs I'm making.